src/test/regress/sql/foreign_key_to_reference_table.sql

```diff
@@ -467,6 +565,23 @@ DROP TABLE referenced_table CASCADE;
 DROP TABLE referencing_table CASCADE;
 DROP TABLE referencing_table2 CASCADE;
 
+-- Check if the same fkeys are created with create_distributed_table
```

Comment:
I think you mean `multiple fkeys` instead of `the same` ? Can we at least get the  `count` from  `table_fkeys_in_workers`?

Reply:
In any case, changed them to count since the order by makes a difference in appveyor..